build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
       jcenter()
     }
     dependencies {
-      classpath "com.android.tools.build:gradle:3.5.0-beta01"
+      classpath "com.android.tools.build:gradle:3.5.0"
```

Comment:
This conflicts with my benchmarks branch, mind removing before merging?

Reply:
Sure.